src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -39,66 +39,251 @@
 # are initially populated.
 BANDIT_PROBABILITY_QUERY = """
 SELECT
-  /* Calculate bandit weights from calculated exponential values. */
-  strategy,
-  strategy_exp_medium_temperature / exp_sum_medium_temperature AS bandit_weight_medium_temperature,
-  strategy_exp_high_temperature / exp_sum_high_temperature AS bandit_weight_high_temperature,
-  strategy_exp_low_temperature / exp_sum_low_temperature AS bandit_weight_low_temperature,
-  run_count
-FROM
-  (SELECT
-    EXP(strategy_avg_edges / medium_temperature) AS strategy_exp_medium_temperature,
-    SUM(EXP(strategy_avg_edges / medium_temperature)) OVER() AS exp_sum_medium_temperature,
-    EXP(strategy_avg_edges / high_temperature) AS strategy_exp_high_temperature,
-    SUM(EXP(strategy_avg_edges / high_temperature)) OVER() AS exp_sum_high_temperature,
-    EXP(strategy_avg_edges / low_temperature) AS strategy_exp_low_temperature,
-    SUM(EXP(strategy_avg_edges / low_temperature)) OVER() AS exp_sum_low_temperature,
-    strategy,
-    run_count
-  FROM
-    (SELECT
-      /* Standardize the new edges data and take averages per strategy. */
-      AVG((new_edges - overall_avg_new_edges) / overall_stddev_new_edges) AS strategy_avg_edges,
+  c.strategy,
+  bandit_weight_medium_temperature,
+  bandit_weight_high_temperature,
+  bandit_weight_low_temperature,
+  c.run_count AS run_count
+FROM (
+  SELECT
+    /* Calculate bandit weights from calculated exponential values. */ strategy,
+    strategy_exp / exp_sum AS bandit_weight_low_temperature,
+    run_count,
+    strategy_selection_method
+  FROM (
+    SELECT
+      EXP(strategy_avg_edges / temperature) AS strategy_exp,
+      SUM(EXP(strategy_avg_edges / temperature)) OVER() AS exp_sum,
       strategy,
-      /* Change temperature parameters here. */
-      .5 as medium_temperature,
-      .75 as high_temperature,
-      .25 as low_temperature,
-      COUNT(*) AS run_count
-    FROM
-      (SELECT
-        fuzzer,
-        CONCAT(s_radamsa, s_max_len, s_ml_rnn, s_vp, s_fork, s_subset, s_recommended_dict) AS strategy,
-        fuzzer_stddev,
-        AVG(new_edges) OVER() AS overall_avg_new_edges,
-        STDDEV(new_edges) OVER() AS overall_stddev_new_edges,
-        new_edges
-      FROM
-        (SELECT
+      run_count,
+      strategy_selection_method
+    FROM (
+      SELECT
+        /* Standardize the new edges data and take averages per strategy. */ AVG((new_edges - overall_avg_new_edges) / overall_stddev_new_edges) AS strategy_avg_edges,
+        strategy,
+        /* Change temperature parameter here. */ .25 AS temperature,
```

Comment:
Can you update the comment to indicate low temperature? Same for other comments.